frontend/packages/core/src/AppLayout/index.tsx

```diff
@@ -15,9 +15,11 @@ const AppGrid = styled(MuiGrid)({
 
 const ContentGrid = styled(MuiGrid)({
   flex: 1,
-  overflow: "hidden",
+  maxHeight: "calc(100vh - 64px)",
```

Comment:
super nit: can we export the height from the header and consume it in both spots?